roles/fio-distributed/tasks/main.yml

```diff
@@ -0,0 +1,55 @@
+---
+- name: Generate fio test
+  k8s:
+    definition:
+      apiVersion: v1
+      kind: ConfigMap
+      metadata:
+        name: fio-test
+        namespace: '{{ operator_namespace }}'
+      data:
+        fiojob: "{{ lookup('template', 'job.fio.j2') }}"
+  when: fiod.servers > 0
+
+- name: Start FIO Server(s)
+  k8s:
+    definition: "{{ lookup('template', 'servers.yaml') | from_yaml }}"
+  when: fiod.servers > 0
+  with_sequence: start=1 count={{fiod.servers}}
+
+- name: Capture pod list
+  k8s_facts:
+    kind: Pod
+    api_version: v1
+    namespace: '{{ operator_namespace }}'
+    label_selectors:
+      - app = fio-benchmark
+  register: server_pods
+  until: "fiod.servers == (server_pods | json_query('resources[].status.podIP')|length)"
+  retries: 10
+  delay: 10
+
+- name: Create IP list
+  set_fact:
+    hosts: |
```

Comment:
`hosts` isn't a safe variable to use and its a bit vague, so perhaps replace with `fio_servers`

Reply:
I don't regard it as unsafe, but noted.

Reply:
That's right unsafe isn't the right word, the variable name doesn't represent the value it will hold, and is generic so can cause conflict. Best to avoid it if possible.